src/sbt-test/sbt-1.4/relax-scalacOptions/test

```diff
@@ -0,0 +1,48 @@
+# TODO: confirm that we shouldn't rewire scalafix/compile in the right task scope to see compileInputsSettings?
+# TODO: earlycompilation/pipelining support
+
+# when the build does not contain any flag relaxed on scalafix invocation
+# check that regular compilation benefits froma prior direct scalafix invocation ...
+-> scalafix --check RemoveUnused
+-> checkLastCompilationCached
+> compile
+> checkLastCompilationCached
```

Comment:
users with no fatal warnings calling scalafix before compile no longer have to compile everything twice